pkgs/top-level/python-packages.nix

```diff
@@ -8647,6 +8647,8 @@ in {
 
   websockify = callPackage ../development/python-modules/websockify { };
 
+  webssh = callPackage ../development/python-modules/webssh { };
```

Comment:
According to the readme, this is meant to also be used as an application. Do you mind adding the following to `pkgs/top-level/all-packages.nix`:
```
  <pkg-name> = with python3Packages; toPythonApplication <python-module>;
```
More information on this can be found [here](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#topythonapplication-function)

Reply:
Fixed, and also disabled the package for Python 2, as the tests will fail with that version (which would have been why [I had them disabled them](https://github.com/NixOS/nixpkgs/pull/122559#discussion_r630021395) when I wrote this package originally).